cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -114,6 +115,69 @@ def test_xlint_fails(self):
         err_cnt = perform_xlint(TEST_DATA_DIR, ['toy'])
         self.assertGreater(err_cnt, 0)
 
+    def test_invalid_asset_overwrite(self):
+        """
+        Tests that an asset with invalid displayname can be overwritten if multiple assets have same displayname.
+        It Verifies that:
+            During import, '/' in displayname of an asset are replaced with underscores '_'.
```

Comment:
@muzaffaryousaf  We can have both `/` and `\` chars in the display_name rather than including a new asset in the `assets.json`. Makes sense?